vitrina/redirections/views.py

```diff
@@ -0,0 +1,11 @@
+from vitrina.redirections.models import Redirections
+from django.shortcuts import redirect
+
+
+def redirection_handler(request, exception):
+    try:
+        existing_url = Redirections.objects.filter(path=request.path)
+        new_url = existing_url.content_object.get_absolute_url
```

Comment:
Please add tests, I don't thing this will work, because `redirect` does not accept callable objects.

https://docs.djangoproject.com/en/3.2/topics/http/shortcuts/#redirect

```suggestion
        new_url = existing_url.content_object
```